docs/example-data.json

```diff
@@ -248,7 +248,7 @@
         "conference_stream_poster_url": "",
         "conference_open_microphone": true,
         "conference_open_video": true,
-        "conference_auto_connect_next_speakers": true,
+        "conference_auto_connect_next_speakers": 1,
```

Comment:
Same as above

Reply:
Quick note: We try to activate everything in the example data, so one does not have to activate everything by hand. But if this is annoying, ... it is ok to have it deactivated ;)